docs/editor/extension-runtime-security.md

```diff
@@ -0,0 +1,63 @@
+
+## Determine extension reliability
+
+Before you install an extension, you can take various steps to determine if it's reliable. The Visual Studio Marketplace provides you with information about the extension to help you make an informed decision:
+
+* **Ratings & Reviews**: Read what others think about the extension.
+
+* **Q & A**: Review existing questions and the level of the publisher's responsiveness. You can also engage with the extension's publisher if you have concerns.
+
+* **Issues, Repository, and License**: Check if the publisher provided these and if they have the support you expect.
+
+* **Verified Publisher**: Use the blue check mark next to the publisher's name and domain name as an extra signal of trust. The check mark indicates that the publisher has proven domain-name ownership to the Marketplace. It also shows that the Marketplace has verified both the existence of the domain name and the good standing of the publisher on the Marketplace for at least six months.
+
+    ![Verified publisher](images/extension-marketplace/bluecheck.png)
+
+## Marketplace protections
+
+The Visual Studio Code Marketplace employs several mechanisms to protect you from malicious extensions:
+
+* **Malware scanning**: The Marketplace runs a malware scan on each extension package that's published to ensure its safety. The scan, which uses several antivirus engines, is run for each new extension and for each extension update. Until the scan is all clear, the extension won't be published in the Marketplace for public usage.
+
+* **Verified publishers**: Publishers can verify (blue check mark) their identity by proving domain ownership. It shows that the publisher has proven domain-name ownership to the Marketplace. It also shows that the Marketplace has verified both the existence of the domain and the good standing of the publisher on the Marketplace for at least six months.
+
+* **Name squatting**: The Marketplace stops extension authors from stealing the names of official publishers, such as Microsoft or RedHat, and popular extensions, like GitHub Copilot.
+
+* **Kill List**: If a malicious extension is reported and verified, or a vulnerability is found in an extension dependency, the extension is removed from the Marketplace and added to a *kill list*. If the extension has been installed, it's automatically uninstalled by VS Code.
+
+* **Extension Signature Verification**: The Visual Studio Marketplace signs all extensions when they're published. VS Code checks this signature when you install an extension to verify the integrity and the source of the extension package.
+
+## Report suspicious extensions
+
+If you do see an extension that looks suspicious, report the extension to the Marketplace team:
+
+1. Open the extension's page in the [Visual Studio Marketplace](https://marketplace.visualstudio.com/vscode).
+
+1. Select the **Report a concern** link at the bottom of the extension **More Info** section.
+
+## Related resources
```

Comment:
There need s to be a link somewhere to  https://code.visualstudio.com/docs/setup/enterprise#_configure-allowed-extensions

The approach would be "Do you want to configure what extensions are allowed in your orgranisation, check out ...."

Reply:
Added as a tip at the end of the Reliability section and also included a link in related resources at the end of the doc.